cliquet/tests/testapp/views.py

```diff
@@ -16,6 +16,7 @@ class Toadstool(resource.ProtectedResource):
     mapping = MushroomSchema()
 
     def get_parent_id(self, request):
+        # Shared collection.
```

Comment:
Same here: when reading the source code, I don't understand why this comment lies here. Since it's an application that could be viewed as an example application, I would rather add some more details in here (yes, it's off-topic with this pull request)